great_expectations/self_check/util.py

```diff
@@ -673,14 +673,15 @@ def get_dataset(
                     df[col] = pd.to_datetime(df[col]).dt.date
 
         if table_name is None:
-            table_name = generate_test_table_name()
+            table_name = generate_test_table_name().lower()
 
         df.to_sql(
             name=table_name,
             con=engine,
             index=False,
             dtype=sql_dtypes,
             if_exists="replace",
+            method="multi",
```

Comment:
Please consider this for defining the table_name and value for method kwarg https://github.com/great-expectations/great_expectations/blob/939be373fb22e10c809aa488e7f1a3521327ed65/great_expectations/self_check/util.py#L1176-L1191